mailhandler/spameater

```diff
@@ -249,6 +249,13 @@ foreach $for (keys %deliveryAddresses) {
    (0,'','','',0,0,0,0,0,0,0,'','',0,'',0,());
 
   ($mprefix,$mword,$mcount,$musername,$mexpiretime) = ('','','','','');
+  
+  if ($from && lc($for) eq lc($from)) {
```

Comment:
I have not been able to find docs for the diff syntax, but I believe that `241a244,251` means: near line 241 **a**dded lines 244-251. If so, that means the code should be just above `# initialize the vars:`. But that will not change the final result of the code. My only suggestion would be to indent the body of the if-statement with 4 spaces to align with the rest of the file.

I will merge this pull request soon after I merge the README pull request.

Reply:
the two commits seem to be in the same branch now, this one with the corrections...